WordPress/src/main/res/values/strings.xml

```diff
@@ -2303,6 +2303,14 @@
     <string name="username_changer_hint">Type to get more suggestions</string>
     <string name="username_changer_title">Change username</string>
 
+    <string name="settings_username_changer_header">You are about to change your username, which is currently %1$s%2$s%3$s.You will not be able to change your username back.</string>
+    <string name="settings_username_changer_confirm_dialog_title">Careful!</string>
+    <string name="settings_username_changer_confirm_dialog_content">You are changing your username to %1$s%2$s%3$s. Changing your username will also affect your Gravatar profile and Intense Debate profile addresses. Confirm your new username to continue.</string>
+    <string name="settings_username_changer_confirm_dialog_positive_action">Change Username</string>
+    <string name="settings_username_changer_progress_dialog">Saving username…</string>
+    <string name="settings_username_changer_toast_content">Your new username is %1$s</string>
```

Comment:
I am realizing that some of the string resources are causing the _lint checker_ to fail possibly because they are being used in ways it's not able to determine that they are in use within the code. Exploring.

Reply:
I think the issue is caused be the "import org.wordpress.android.R.string". Apparently, lint doesn't expect that and it's looking just for `R.string.xyz`.